requirements.txt

```diff
@@ -6,5 +6,7 @@ pyyaml>=3.12
 cloudpickle>=0.4.0
 distributed>=1.24.2
 dask[dataframe]>=1.1.0
+koalas>=1.0.1
+pyspark>=3.0.0
```

Comment:
koalas and pyspark should probably be optional, see https://github.com/FeatureLabs/featuretools/issues/1037